lib/storage/client.js

```diff
@@ -1465,5 +1465,38 @@ class Client extends CryptoConsumer {
     )
     return await validateResponseAsJSON(response)
   }
+
+  /**
+   * removeSharedRecordWithGroup
+   *
+   * @param {string} groupId   UUID assigned by TozStore, used to identify a group.
+   * @param {string} recordType Record type to be unshared with the group.
+   *
+   * @return {Promise<Record>}
+   */
+  async removeSharedRecordWithGroup(groupId, recordType) {
```

Comment:
potentially better to have this be "revokeRecordWithGroup" because it'll match our other sdk methods (share/revoke)